sdk/core/core/src/az_aad.c

```diff
@@ -100,7 +100,7 @@ AZ_NODISCARD az_result _az_aad_request_token(_az_http_request* request, _az_toke
   AZ_RETURN_IF_FAILED(az_http_request_append_header(
       request,
       AZ_SPAN_FROM_STR("Content-Type"),
-      AZ_SPAN_FROM_STR("application/x-www-url-form-urlencoded")));
```

Comment:
Ha, this is where their docs and postman sample they provide disagree (https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow). But I'd go with your change, assuming that you verified that it works.

Reply:
yeah, we got an error doing it the other way.
I see their doc with this value as well: https://github.com/Azure/azure-sdk-for-c/pull/573

So, it was just a typo somewhere